cli/js/truncate.ts

```diff
@@ -1,6 +1,14 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 import { sendSync, sendAsync } from "./ops/dispatch_json.ts";
 
+export interface TruncateOptions {
+  /** Permissions to use if creating the file (defaults to `0o666`, before
+   * the process's umask).
+   * It's an error to specify mode when create is set to `false`.
+   * Does nothing/raises on Windows. */
+  mode?: number;
+}
```

Comment:
Ideally we'd add a unit test of this too...

Reply:
Right, but see my comment above... I realized after pushing this it couldn't be tested until the `create` option is added to truncate too (I was going to do that after you all had a chance to review this permission-related batch of PRs).